Src/BootCamp.Chapter/Computer/Motherboard.cs

```diff
@@ -2,5 +2,15 @@
 {
     public class Motherboard
     {
+        private string _manufacturersId;
+        public Motherboard(string manufacturersId)
+        {
+            _manufacturersId = manufacturersId;
+        }
+
+        public string GetManufacturersId()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider making method 'GetManufacturersId' a property.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5121609)